components/eam/src/chemistry/mozart/mo_gas_phase_chemdr.F90

```diff
@@ -449,7 +455,14 @@ subroutine gas_phase_chemdr(lchnk, ncol, imozart, q, &
     !-----------------------------------------------------------------------      
     !        ... Xform from mmr to vmr
     !-----------------------------------------------------------------------      
-    call mmr2vmr( mmr, vmr, mbar, ncol )
+!    call mmr2vmr( mmr, vmr, mbar, ncol )
+!AH -- added ST80
```

Comment:
Remove signatures/initials; replace with more meaningful documentation if necessary. Might be worthwhile to put in a comment explaining what this is and why.